module/documents/chat-message.mjs

```diff
@@ -85,6 +85,12 @@ export function addChatMessageContextOptions(html, options) {
       condition: canApply,
       callback: li => applyChatCardDamage(li, -1)
     },
+    {
+      name: game.i18n.localize("DND5E.ChatContextTempHP"),
+      icon: '<i class="fas fa-user-clock"></i>',
```

Comment:
I'm not certain a clock is the most intuitive icon here.

Reply:
The clock was meant to symbolize time. Temporary hit points expire at some point, so using the clock seemed appropriate. I also looked through the FA icons, the `user` ones anyways, and didn't see anything better IMO.

Reply:
I think that's a much better choice!

Reply:
Just that it's not consistent with the existing icons, that was my point. But it's not a hill I'm willing to die on

Reply:
That's a good point, the other four icons show a person silhouette while this does not.  Can you maybe take a screenshot of how each of them looks?  it's possible that using a clock is indeed better just for the consistency.

Reply:
@shemetz Here's a screenshot with the `fa-user-clock` icon:

![image](https://user-images.githubusercontent.com/19502084/187642161-8040bd15-e573-47f5-8273-3c9cec0fec93.png)

And one with the `fa-shield-heart` icon:

![image](https://user-images.githubusercontent.com/19502084/187642511-d853258a-b1f3-48b8-8043-123e09403583.png)

Reply:
so yeah, I think user-clock does fit the best for this.  the one for Double Damage is also slightly different but... oh well.

Reply:
True, the double damage doesn't have an icon in the corner but it's clever IMO

Reply:
I'd agree that a clock is reasonably intuitive to represent something that is 'temporary', and is consistent with our other `fa-user` icons. We can change it later very easily if people still feel strongly.